### ISSUE TITLE
"Cannot find symbol for node" when using labeled tuples

### ISSUE THREAD

**Issue by guansss** [timocov/dts-bundle-generator#143]
**Bug report**

Thanks for making this helpful tool. However, I've encountered a problem while using labeled tuples that TypeScript 4.0 introduced. Thought it's not an effort to revert those labeled tuples back to normal tuples in my project, I would like to point out this issue.

**Input code**

```ts
// index.ts
export const a: [x: number, y: number] = [1, 2];
```

**Expected output**

Compiles without error.

**Actual output**

```
Compiling input files...

Error: Cannot find symbol for node: x
```

**Additional context**

Versions:
typescript 4.1.3
dts-bundle-generator 5.5.0

**Comment by timocov** [comment]
Thanks for reporting! It's definitely a bug and labelled tuples should be handled correctly.

**Comment by timocov** [comment]
The fix is just deployed with 5.6 version.
